src/kernel-install/kernel-install

```diff
@@ -182,7 +180,7 @@ case "$COMMAND" in
             [ "$KERNEL_INSTALL_VERBOSE" -gt 0 ] && echo "+$f remove $KERNEL_VERSION $ENTRY_DIR_ABS"
             "$f" remove "$KERNEL_VERSION" "$ENTRY_DIR_ABS"
             err=$?
-            [ $err -eq $skip_remaining ] && break
+            [ $err -eq 77 ] && break
```

Comment:
Hmm, I think it was better to have the variable name as documentation.

Reply:
Sure, dropped